app/code/Magento/Reports/view/adminhtml/templates/store/switcher/enhanced.phtml

```diff
@@ -3,44 +3,43 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
 
-/**
- * @deprecated
- */
+use Magento\Backend\Block\Store\Switcher;
+use Magento\Framework\Escaper;
 
-/**
- * @see \Magento\Backend\Block\Store\Switcher
- */
+/** @var Escaper $escaper */
+/** @var Switcher $block */
 
 /**
- * @var $block \Magento\Backend\Block\Store\Switcher
+ * @deprecated
```

Comment:
It seems we should keep @see tag here, because template is deprecated

```suggestion
 * @deprecated
 *
 * @see \Magento\Backend\Block\Store\Switcher
```

Reply:
Good idea, @andrewbess , actioned 🙂